app/src/main/java/eu/faircode/email/FragmentMessages.java

```diff
@@ -759,6 +759,13 @@ public void onChildDraw(
             }
         }
 
+        @Override
+        public void onSelectedChanged(@Nullable RecyclerView.ViewHolder viewHolder, int actionState) {
+            super.onSelectedChanged(viewHolder, actionState);
+            final boolean swiping = actionState == ItemTouchHelper.ACTION_STATE_SWIPE;
+            swipeRefresh.setEnabled(!swiping);
```

Comment:
This is a nice addition. Did you test if it works properly?

Reply:
Tested and working properly, but should be disabled when pull to refresh was disabled (see commit).

Reply:
Good catch, thanks!
And yes it works well for me too.